src/ssl.c

```diff
@@ -6201,6 +6334,13 @@ int wolfSSL_CertManagerVerifyBuffer(WOLFSSL_CERT_MANAGER* cm, const byte* buff,
         ret = CheckCertCRL(cm->crl, cert);
 #endif
 
+#ifndef NO_WOLFSSL_CM_VERIFY
+    /* if verify callback has been set */
+    if (cm->verifyCallback) {
+        ret = CertManager_DoVerifyCallback(cm, ret, cert);
```

Comment:
This is called for each CM verify.  Your commit comment says for each cert loaded.  What about wolfSSL_CertManagerLoadCA()?  Is that not covered by design?

Reply:
I can revise the wording in the commit. The API currently relies on the application to call `wolfSSL_CertManagerVerify` for each loaded certificate.